.github/README.md

```
@@ -554,17 +553,17 @@ Read the [Contribution Guidelines](https://github.com/YASHSHARMAOFFICIALLY/Argoy
 
 ### Commit Convention
 
-[Click Here](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/blob/Bug-Resolve/main-README/docs/COMMIT_CONVENTION.md) to follow the commit conventions while committing code to this Project.
+[Click Here](https://github.com/YASHSHARMAOFFICIALLY/Argoya_Assam/blob/main/docs/COMMIT_CONVENTION.md) to follow the commit conventions while committing code to this Project.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)